python/lsst/shapelet/shapeletFunction/shapeletFunctionContinued.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import, division, print_function
+
+from .shapeletFunction import ShapeletFunction
+
+from lsst.utils import continueClass
+
+__all__ = []
+
+@continueClass
+class ShapeletFunction:
+    def __reduce__(self):
+        return (ShapeletFunction, (self.getOrder(), self.getBasisType(),
+            self.getEllipse(), self.getCoefficients()))
```

Comment:
That said...would it make any sense to turn this into C++ code? Or is that just too much work for right now (since this ticket is huge). Perhaps consider filing a ticket to define `__reduce__` in C++ pretty much everywhere? It seems a shame to add this whole subdirectory structure when C++ could do the job.